packages/Algernon.package/ALGResultItemMorph.class/instance/icon..st

```diff
@@ -1,3 +1,3 @@
-accessing
+thumbnail
```

Comment:
just fyi - some of those classifications were made bc there is a linting test that checks if the categories match the category of the methods in the super classes. Appereantly there is somewhere very up in the hirarchy some weird categorization so that I decided to not run this lint in the end. But yes, completely right I missed to revoke some of those changes - so thanks 👍